user-docs/lsst-specific-content-style-guide.rst

```diff
@@ -0,0 +1,31 @@
+.. _user-doc-style-lsst:
+
+#########################
+LSST-specific style guide
+#########################
+
+The :doc:`LSST user documentation style guide </user-docs/index>` is built upon the `Google Developer Style Guide`_.
+This page lists our exceptions and modifications to those guidelines.
+
+Use the imperative mood for Python and C++ function and method summary sentences
+--------------------------------------------------------------------------------
+
+The Google Developer Style Guide recommends `using the present tense for function and method summaries <https://developers.google.com/style/api-reference-comments#methods>`__.
+For example:
+
+   Gets the value.
+
+For LSST DM Python and C++, you should instead use the **imperative mood** in this context:
+
+   Get the value.
```

Comment:
I would suggest reversing the order of these examples:

```
For LSST DM Python and C++, use imperative:
  ...
but if you are documenting an unrelated technology, for example HTTP API endpoints, use present tense, as recommended by the Google guide:
  ...
(discussion of rationale)
```

That way, the first thing a style guide user sees is the "correct" LSST usage for most cases.

Reply:
Good idea